src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -481,3 +431,80 @@ export const AddressVerificationFlowQueryRenderer: React.FC<{
     />
   )
 }
+
+const modalTitles: Record<VerificationPath, string | null> = {
+  [VerificationPath.SUGGESTIONS]: "Confirm your delivery address",
+  [VerificationPath.REVIEW_AND_CONFIRM]: "Check your delivery address",
+  [VerificationPath.ERROR_IMMEDIATE_CONFIRM]: null,
+  [VerificationPath.IMMEDIATE_CONFIRM]: null,
+} as const
+
+const fallbackFromFormValues = (address: AddressValues): AddressOption => {
+  return {
+    key: USER_INPUT,
+    recommended: false,
+    lines: [
+      address.addressLine1,
+      address.addressLine2 || null,
+      `${address.city}, ${address.region} ${address.postalCode}`,
+      address.country,
+    ].filter(Boolean) as string[],
+
+    address: {
+      addressLine1: address.addressLine1,
+      addressLine2: address.addressLine2,
+      region: address.region,
+      country: address.country,
+      city: address.city,
+      postalCode: address.postalCode,
+    },
+  }
+}
+
+const useAddressVerificationTracking = () => {
```

Comment:
non-blocking, but just wanted to point out that moving the tracking to a different place may cause dupes. As long as tracking only runs once within a useEffect() there shouldn't be any issues.

Reply:
I think we should be ok here - because tracking calls have been removed from the previous version (`useEffect` with a big chain of if/elses that calls side effects) the code is now more explicitly structured to run the tracking calls only on initial load and when a user selects an address. So my hope is that this will actually be safer.